WebApi/Services/CreateReviewsJobProcessor.cs

```diff
@@ -15,20 +15,17 @@ public CreateReviewsJobProcessor(IConfiguration configuration, AppDbContext dbCo
         _dbContext = dbContext;
     }
 
-    public async Task<string?> ProcessJobAsync(Job job, CancellationToken cancellationToken)
+    public async Task<string?> ProcessJobAsync(Job job, CancellationToken stoppingToken)
```

Comment:
so the type is _CancellationToken_, but the name is _stoppingToken_. Why is that? Does the cancellationToken generally serve a different purpose than just cancellation? At least change the underlying properties to reflect the updated name then

Reply:
Read the documentation of BackgroundService.

Reply:
I meant the object's properties. I do see that they name it stoppingToken in the documentation, alright then 👍